daisy/README.md

```diff
@@ -1,28 +1,128 @@
+relationships
 
-Daisy is a GCE workflow tool. Development is ongoing.
-https://godoc.org/github.com/GoogleCloudPlatform/compute-image-tools/daisy
+# What is Daisy?
+Motivated by Google-internal image build workflows, Daisy is a solution for
+running complex, multi-step procedures on and involving GCE.
 
-## Table of contents
+The current Daisy stepset includes support for creating/deleting GCE resources,
+waiting for signals from GCE VMs, streaming GCE VM logs, uploading local files
+to GCE and GCE VMs, and more.
 
-  * [Workflow Sources](#workflow-sources)
-  * [Workflow Steps](#workflow-steps)
-    * [AttachDisks](#attachdisks)
-    * [CreateDisks](#createdisks)
-    * [CreateImages](#createimages)
-    * [CreateInstances](#createinstances)
-    * [DeleteResources](#deleteresources)
-    * [RunTests](#runtests)
-    * [SubWorkflow](#subworkflow)
-    * [WaitForInstancesSignal](#waitforinstancessignal)
-    * [WaitForInstancesStopped](#waitforinstancesstopped)
+## Table of contents
+  * [Setup](#setup)
+  * [Running Daisy](#running-daisy)
+  * [Workflow Config Overview](#workflow-config-overview)
+    * [Autovars](#autovars)
+    * [Sources](#sources)
+    * [Steps](#steps)
+      * [AttachDisks](#attachdisks)
+      * [CreateDisks](#createdisks)
+      * [CreateImages](#createimages)
+      * [CreateInstances](#createinstances)
+      * [DeleteResources](#deleteresources)
+      * [RunTests](#runtests)
+      * [SubWorkflow](#subworkflow)
+      * [WaitForInstancesSignal](#waitforinstancessignal)
+      * [WaitForInstancesStopped](#waitforinstancesstopped)
   * [Dependency Map](#dependency-map)
 
-## Workflow Sources
+## Setup
+Daisy is available for Windows, macOS, and Linux distros. We are in the process
+of releasing precompiled binaries.
+
+Until then, Daisy is easily buildable with the Golang SDK.
+```shell
+go get github.com/GoogleCloudPlatform/compute-image-tools/daisy
+```
+This will place the Daisy binary in $GOPATH/bin.
+
+## Running Daisy
+The basic use case for daisy looks like:
+```shell
+daisy [path to workflow config file]
+```
+
+Many workflow config fields and variables can be overridden, for example:
+```shell
+daisy wf.json -project my-other-project \\
+              -zone other-zone \\
+              -variables foo=bar,baz=gaz
+```
+
+For additional information about Daisy flags, use `daisy -h`.
+
+## Workflow Config Overview
+A workflow config is written in JSON and contains the following fields:
+
+| Field Name | Type | Description |
+|-|-|-|
+| Name | string | The name of the workflow. Must be between 1-20 characters and match regex **[a-z]\([-a-z0-9]\*[a-z0-9])?**|
+| Project | string | The GCE and GCS API enabled GCP project in which to run the workflow. |
+| Zone | string | The GCE zone in which to run the workflow. |
+| OAuthPath | string | A local path to JSON credentials for your Project. These credentials should have full GCE permission and read/write permission to GCSPath.|
```

Comment:
Add a note that if this is not set common credential locations will be searched. So a user that has run `gcloud init` doesn't need to set this. Also if running daisy on a GCE instance it will use the default service account